src/main/webapp/pages/analysis/visualizations/phylocanvas-metadata.html

```diff
@@ -73,10 +73,10 @@
         <div class="spaced-bottom" id="phylocanvas"></div>
       </script>
       <script type="text/ng-template" id="metadata-component.tmpl.html">
-        <div class="spaced-bottom" ng-repeat="term in $ctrl.terms"> <input
-        type="checkbox" bs-switch="" ng-model="term.selected" switch-label="{{
-          term.term
-        }}" switch-label-width="200" ng-change="$ctrl.onSelectionChange(term)"/>
+        <div class="spaced-bottom" ng-repeat="term in $ctrl.terms">
+          <div class="checkbox">
```

Comment:
Think it would be worth it to add a small header to that slideout?  It seems very bare with just the checkboxes for some reason.
![image](https://user-images.githubusercontent.com/8951708/86405642-51ef7880-bc77-11ea-8463-9ce242f75342.png)

Reply:
Updated in 4277646